src/styles/blocks/card.scss

```diff
@@ -0,0 +1,38 @@
+.card {
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px;
```

Comment:
The `border` shorthand property should include values for `border-width`, `border-style`, and `border-color`. For example, `border: 1px solid #000;`. Make sure to define all these properties or use individual properties if you intend to only set the width.